### PR TITLE
Open Shortcuts app URLs with a confirmation dialog

### PR DESCRIPTION
**Pull request by miasma13**
Task/Issue URL: https://app.asana.com/0/414235014887631/1203051209824446/f
Tech Design URL:
CC:

**Description**:
We define platform schemes that prompt user for confirmation as opening those URLs redirects user to another app. iOS provides a `shortcuts` scheme for launching the Shortcuts app which should be handled in the same way.

**Steps to test this PR**:
1. Create on a simulator or device a shortcut with a predefined name (e.g. "Test shortcut")
2. On a static webpage or jsfiddle.net create a following link (provide the right name as the `name` parameter): 
`<a href="shortcuts://run-shortcut?name=Test%20shortcut">LINK</a>`
3. Tap the link to run the shortcut 4. 

**Device Testing**:
* [ ] iPhone
* [ ] iPad

**OS Testing**:
* [ ] iOS 14
* [x] iOS 15
* [x] iOS 16

### CODE REVIEW

**Comment by bwaresiak** [comment]
Small note: it probably would be good to add this test case to https://github.com/duckduckgo/privacy-test-pages

File: Core/SchemeHandler.swift [hunk 1]
```diff
@@ -68,7 +69,7 @@ public class SchemeHandler {
         }
 
         switch PlatformScheme(rawValue: schemeString) {
-        case .sms, .mailto, .itms, .itmss, .itunes, .itmsApps, .itmsAppss:
+        case .sms, .mailto, .itms, .itmss, .itunes, .itmsApps, .itmsAppss, .shortcuts:
```
Comment (bwaresiak) [inline Core/SchemeHandler.swift]: There is also `shortcuts-production` and `workflow`

**Review by bwaresiak — APPROVED** [review]
LGTM, added one comment inline, good to merge when ready.

File: Core/SchemeHandler.swift (hunk 1 above)
Comment (miasma13) [reply Core/SchemeHandler.swift]: Great find, thanks.

**Pull request merged**